src/RockBridge.hpp

```diff
@@ -1,68 +0,0 @@
-//======================================================================================
-// Brazilian Institute of Robotics 
-// Authors: Thomio Watanabe
-// Date: December 2014
-//====================================================================================== 
-#ifndef _ROCK_BRIDGE_HPP_
-#define _ROCK_BRIDGE_HPP_
-
-// Gazebo headers
-#include <boost/bind.hpp>
-#include <gazebo/gazebo.hh>
-#include <gazebo/physics/physics.hh>
-#include <gazebo/common/common.hh>
-#include <gazebo/common/Plugin.hh>
-// Rock Headers
-#include <rtt/types/TypekitRepository.hpp>
-#include <rtt/transports/corba/TaskContextServer.hpp>
-#include <rtt/TaskContext.hpp>
-#include <rtt/Activity.hpp>
-//#include <rtt/extras/SlaveActivity.hpp>
-
-#define GROUND 0
```

Comment:
__NEVER__ use #define to create constants in C++ unless you are __absolutely very sure__ it is the only way. And if you really need to, then define constants that are so long and complex that nobody will ever use them by mistake (i.e. not GROUND and UNDERWATER). #define are not scoped, they apply to every piece of code that includes your header and lead to very weird errors if the unsuspecting user of your header uses the same symbol. 

For integer constants, use static const int within a class context, e.g.

```
class RockBridge
{
public:
    static const int ENVIRONMENT_GROUND = 0;
    static const int ENVIRONMENT_UNDERWATER = 1;
   ...
};
```

You can also create yourself an enum __within your code's namespace__. Use specific names as well (e.g. the ENVIRONMENT_ prefix above).